src/main/Validator.java

```diff
@@ -0,0 +1,82 @@
+package main;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Validator {
+
+	private List<BankAccount> accounts = new ArrayList<>();
+
+	public boolean checkValidAccountType(final String type) {
+		List<String> validAccounts = Arrays.asList("checking", "savings", "cd");
+
+		if (validAccounts.contains(type)) {
+			return true;
+		}
+		return false;
+	}
+	
+	public boolean checkValidAccountId(final String id) {
+		if(accounts.contains(id)) {
+			return true;
+		}
+		return false;
+		
+	}
+	
+	public boolean checkValidAmount(final String amount) {
+		if(Integer.parseInt(amount) > 0) {
+			return true;
+		}
+		return false;
+		
+	}
+
+	public boolean validateCommandsGiven(final String command) {
+		String[] commandsGiven = new String[0];
+
+		if (command != "") {
+			commandsGiven = command.split(" ");
+			if (commandsGiven[0] == "create") {
+				return createCheck(commandsGiven);
+
+			} else if (commandsGiven[0] == "deposit") {
+				return depositeCheck(commandsGiven);
+
+			} else if (commandsGiven[0] == "withdraw") {
+				return withdrawCheck(commandsGiven);
+
+			}
+		}
+		return false;
+	}
+
+	public boolean createCheck(final String[] commands) {
```

Comment:
I have no idea what this and the methods below are doing by reading the name of the methods